objects.go

```diff
@@ -323,10 +323,11 @@ func NewNetworkContainer(netview, cidr string, isIPv6 bool, comment string, ea E
 }
 
 type NetworkContainerNextAvailable struct {
-	IBBase  `json:"-"`
-	Network *NetworkContainerNextAvailableInfo `json:"network"`
-	Comment string                             `json:"comment"`
-	Ea      EA                                 `json:"extattrs"`
+	IBBase      `json:"-"`
+	Network     *NetworkContainerNextAvailableInfo `json:"network"`
+	NetviewName string                             `json:"network_view,omitempty"`
```

Comment:
From WAPI, If we specify a network view for automatic network selection, we should also add a network_view field in the object to be inserted with the same network view because the network view for automatic network selection is not used for the actual object insertion.